platform/core/src/redux/reducers/viewports.js

```diff
@@ -88,6 +90,10 @@ const viewports = (state = defaultState, action) => {
       } else {
         return defaultState;
       }
+    case FORCE_VIEWPORT_UPDATE:
+      return Object.assign({}, state, {
+        updateHash: new Date().getUTCMilliseconds().valueOf(),
+      });
```

Comment:
This is an interesting approach. What are you changing that you need to force a viewport update in this way?

Reply:
@deprecated